docs/features/asc1/stateful/hello_world.md

```diff
@@ -23,33 +23,34 @@ Every Algorand stateful application is comprised of at least two _programs_: *ap
 The _approval program_ is the main application logic for the smart contract. 
 
 #### Clear State Program
-The _clear state_ program is used to retire an application, its global state and remove local state from the user’s account. 
+The _clear state_ program is used to retire an application, its global state, and remove local state from the user’s account. 
```

Comment:
Suggestion: The _clear state_ program is used to retire an application, remove its global state from the creator's account and remove local state from any opt-in users' accounts.